docs/src/commands/build.md

```diff
@@ -32,6 +32,16 @@ wasm-pack build --out-dir out
 The above command will put your build artifacts in a directory called `out`, instead
 of the default `pkg`.
 
+## Output file names
+
+Set output file names. Defaults to package name.
+
+```
+wasm-pack build --out-name index
+```
+
+This command will build the package with file names like `index.js`, `index.d.ts`, and so on.
```

Comment:
i'd also included what the wasm file is called here (this covers almost all the files and since the wasm one is pretty important, i'd include that here.)

Reply:
Updated with more detailed information